valentyusb/usbcore/sm/send.py

```diff
@@ -287,7 +288,7 @@ def stim(dut):
 
         run_simulation(
             dut, stim(dut),
-            vcd_name="vcd/test_token_decode_%s.vcd" % self.id(),
+            vcd_name=self.make_vcd_name(),
```

Comment:
In future I think we add the whole run_simulation call to one of the base classes?

Reply:
Yes, that's my plan if only to hook the process of creating vcd files so that we can build a gtkwave config for it as well.  For the VCD names a simple generator method seemed easier than trying to change it in `**kwargs**` when running the simulation.

Ewen

PS: have swapped comments in test/common.py to docstrings; good suggestion.